dev/sg/dependencies/shared.go

```diff
@@ -136,7 +136,7 @@ func categoryProgrammingLanguagesAndTools(additionalChecks ...*dependency) categ
 			},
 			{
 				Name:        "pnpm",
-				Description: "Run `asdf install pnpm` or `npm i -g pnpm`",
+				Description: "Run `asdf plugin add pnpm && asdf install pnpm` or `npm i -g pnpm`",
```

Comment:
```suggestion
				Description: "Run `asdf plugin add pnpm && asdf install pnpm`",
```
The `npm i -g pnpm` is what breaks most peoples installs currently... only noticed now that we recommend it 😶